ckanext/security/controllers.py

```diff
@@ -81,40 +80,47 @@ def login(self):
         Ajax call to test username/password/mfa code
         """
         tk.response.headers.update({'Content-Type': 'application/json'})
-        if request.method != 'POST':
-            tk.response.status_int = 405
-            return
 
-        identity = request.params
-        if not ('login' in identity and 'password' in identity):
-            tk.response.status_int = 422
-            return
+        try:
+            if request.method != 'POST':
+                tk.response.status_int = 405
+                return
```

Comment:
With these returns, is this still somewhere that could result in outputting a page after setting response type to json?